ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -812,7 +812,7 @@ tests:
       TEST_ADDITIONAL: Network_Observability
     test:
     - chain: openshift-e2e-test-qe-destructive
-    workflow: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips
```

Comment:
I guess we need to update the job name with fips as well if the workflow is updated, right?

Reply:
Good point. Seems E2E_RUN_TAGS also need to be updated.

Reply:
looks good to me